packages/web-app-activities/src/components/ActivityItem.vue

```diff
@@ -0,0 +1,110 @@
+<template>
+  <div class="oc-flex oc-flex-middle activity-item">
+    <div class="oc-flex oc-flex-middle">
+      <oc-avatar :width="36" :user-name="activity.template.variables?.user?.displayName" />
+      <span class="oc-ml-s" v-text="activity.template.variables?.user?.displayName" />
+    </div>
+    <div>activity unknown</div>
+    <div class="oc-text-truncate">
+      <resource-list-item v-if="resource" :resource="resource" :is-resource-clickable="false" />
+      <div
+        v-if="resourceNotAccessible"
+        v-oc-tooltip="$gettext('The resource is unavailable, it may have been deleted.')"
+        class="oc-text-muted oc-flex oc-flex-middle oc-p-xs"
+      >
+        <oc-icon name="eye-off" />
+        <span class="oc-ml-s" v-text="activity.template.variables?.resource?.name" />
+      </div>
+    </div>
+    <div><span v-text="recordedDateTime" /></div>
+  </div>
+</template>
+
+<script lang="ts">
+import { computed, defineComponent, onMounted, PropType, ref, unref } from 'vue'
+import { Activity as GraphActivity, User } from '@ownclouders/web-client/graph/generated'
+import { DateTime } from 'luxon'
+import {
+  formatDateFromDateTime,
+  formatRelativeDateFromDateTime,
+  ResourceListItem,
+  useClientService
+} from '@ownclouders/web-pkg'
+import { useGettext } from 'vue3-gettext'
+import { Resource, SpaceResource } from '@ownclouders/web-client'
+
+//TODO: Use original type Activity from web-client when it's corrected
+type Activity = GraphActivity & {
+  template: {
+    variables?: {
+      space?: SpaceResource
+      resource?: Resource
+      user?: User
+    }
+  }
+}
```

Comment:
@kulmann ewwwww... but I don't find a better way around that

Reply:
hm... fine as a workaround until https://github.com/owncloud/ocis/issues/10151 is fixed. Could you add a `FIXME` comment with a reference to the issue?